### PR TITLE
Strip out blank results for facilitator specific results

### PR DESCRIPTION
**Pull request by mehalshah**
Use he to escape free text results and strip out blank results
https://trello.com/c/69kfFZLW/634-malformatting-of-apostrophes-in-survey-responses
https://trello.com/c/wpTHYtVz/635-hide-blank-whitespace-only-free-response-answers

### CODE REVIEW

File: apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx [hunk 1]
```diff
@@ -114,7 +115,7 @@ export default class Results extends React.Component {
                     {
                       this.state.facilitatorIds.map((id, j) => (
                         <td key={j}>
-                          {this.props.thisWorkshop[session]['facilitator'][question_key][id]}
+                          {he.decode(this.props.thisWorkshop[session]['facilitator'][question_key][id])}
```
Comment (aoby) [inline apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx:118]: Is this related to stripping out blank results?

Comment (mehalshah) [reply apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx:118]: Nope - this is for https://trello.com/c/69kfFZLW/634-malformatting-of-apostrophes-in-survey-responses
